samples/central-login/.env.example

```diff
@@ -1,6 +1,5 @@
 AM_URL=$AM_URL
 REALM_PATH=$REALM_PATH
-REDIRECT_URI=$REDIRECT_URI
```

Comment:
Do we not use this in blog posts?

Reply:
It doesn't seem to be used anywhere in the sample - is it required?

Reply:
It's not used in the central-login sample, though one of the files ([webpack.config.js](https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/samples/central-login/webpack.config.js#L7)), references it. Omitting this from the .env doesn't seem to cause any issues with the central-login app in any case, so it should probably be removed from ([webpack.config.js](https://github.com/ForgeRock/forgerock-javascript-sdk/blob/develop/samples/central-login/webpack.config.js#L7)).

Reply:
I've removed it from there also, thanks David.